snippets/microshift-upload-cont2-mirror-script.adoc

```diff
@@ -1,3 +1,5 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
snippet?

Reply:
Same comment as other PR, thanks! (And sorry for not looking myself closer)